src/mumble/Global.cpp

```diff
@@ -107,6 +107,7 @@ Global::Global(const QString &qsConfigPath) {
 	bCenterPosition = false;
 	bPosTest        = false;
 	bInAudioWizard  = false;
+	bInConfigUI     = false;
```

Comment:
With the new Mumble style guide we do not use the type indicator prefix on member variables anymore. (We only apply the new Mumble style guide to new code and code that gets changed). 
So your new member should be ``inConfigUI`` instead of ``bConfigUI``.